templates/zerver/help/logging-out.md

```diff
@@ -6,21 +6,13 @@
 1. Click on the **gear** (<i class="fa fa-cog"></i>) icon in the upper
 right corner.
 
-2. Click **Log out**.
+1. Click **Log out**.
 
-{tab|ios}
+{tab|mobile}
 
-1. Tap the menu (<i class="fa fa-reorder"></i>) icon in the top left
-corner of the screen.
+1. {!mobile-profile-menu.md!}
 
-2. Tap **Log out**.
-
-{tab|android}
-
-1. Tap the overflow ( <i class="fa fa-ellipsis-v"></i> ) icon in
-the top right corner of the screen.
-
-2. Tap **Log out**.
+1. Tap **Log out**.
 
```

Comment:
Let's add an instruction for the confirmation dialog. Something like:

> 1. Tap **Log out** to confirm.

This one will have to be updated following up on the changes discussed in https://chat.zulip.org/#narrow/stream/48-mobile/topic/confirm.2Fcancel.20modals

Reply:
Sounds good to me!